internal/stack/serverlessresources.go

```diff
@@ -39,20 +43,35 @@ func applyServerlessResources(profile *profile.Profile, stackVersion string, con
 
 	resourceManager := resource.NewManager()
 	resourceManager.AddFacter(resource.StaticFacter{
-		"agent_version": stackVersion,
-		"agent_image":   appConfig.StackImageRefs(stackVersion).ElasticAgent,
-		"username":      config.ElasticsearchUsername,
-		"password":      config.ElasticsearchPassword,
-		"kibana_host":   config.KibanaHost,
-		"fleet_url":     config.Parameters[paramServerlessFleetURL],
+		"agent_version":      stackVersion,
+		"agent_image":        appConfig.StackImageRefs(stackVersion).ElasticAgent,
+		"logstash_image":     appConfig.StackImageRefs(stackVersion).Logstash,
```

Comment:
Why not keep the same approach using `image: ${ELASTIC_AGENT_IMAGE_REF}` as in local stack? 

Those variables are injected by elastic-package when running `elastic-package stack up`. Maybe to keep consistency I would keep with those envs.

cc @jsoriano

Reply:
I kept this similar to `agent_image` . It was already added

Reply:
> Why not keep the same approach using `image: ${ELASTIC_AGENT_IMAGE_REF}` as in local stack?

I think `StackImageRefs` already take into account the environment variables.